src/app/workspace/[workspaceId]/header.tsx

```
@@ -28,188 +29,210 @@ import { cn } from "@/lib/utils";
+                <DialogDescription className="text-sm text-muted-foreground">
+                  Select a workspace to switch to.
+                </DialogDescription>
+              </DialogHeader>
+
+              <div className="flex flex-col gap-y-3 p-5">
+                {workspaces?.map((item) => (
+                  <button
+                    className="flex w-full cursor-pointer items-center gap-x-4 rounded-[10px] border bg-card px-4 py-3 hover:bg-accent/10 transition-standard hover:translate-x-1 group"
+                    key={item._id}
+                    onClick={() => onWorkspaceClick(item._id)}
+                  >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)